sp041_20191205T1757-instrument-scalar_v2.cdl

```diff
@@ -0,0 +1,417 @@
+		DEPTH:standard_name = "depth" ;
+		DEPTH:units = "m" ;
+		DEPTH:ancillary_variables = "DEPTH_QC" ;
+		DEPTH:positive = "down" ;
+        DEPTH:sensor = "/CTD";
+	byte DEPTH_QC(N_MEASUREMENTS) ;
+		DEPTH_QC:_FillValue = 0b ;
+		DEPTH_QC:long_name = "quality flag" ;
+	float TEMP(N_MEASUREMENTS) ;
+		TEMP:_FillValue = -9999.9 ;
+		TEMP:long_name = "Sea Water Temperature" ;
+        TEMP:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/TEMP/";
+		TEMP:standard_name = "sea_water_temperature" ;
+		TEMP:units = "Celsius" ;
+		TEMP:ancillary_variables = "TEMP_QC" ;
+		TEMP:valid_max = 40. ;
+		TEMP:valid_min = -5. ;
+		TEMP:coverage_content_type = "physicalMeasurement" ;
+		TEMP:coordinates = "time lon lat depth" ;
+		TEMP:comment = "Sea temperature in-situ ITS-90 scale" ;
+        TEMP:sensor = "/CTD";
+	byte TEMP_QC(N_MEASUREMENTS) ;
+		TEMP_QC:_FillValue = 0b ;
+		TEMP_QC:long_name = "quality flag" ;
+	float PSAL(N_MEASUREMENTS) ;
+		PSAL:_FillValue = NaN ;
+		PSAL:long_name = "Sea Water Salinity" ;
+        PSAL:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/PSAL/";
+		PSAL:standard_name = "sea_water_practical_salinity" ;
+		PSAL:units = "1" ;
+		PSAL:ancillary_variables = "PSAL_QC" ;
+		PSAL:valid_max = 40. ;
+		PSAL:valid_min = 0. ;
+		PSAL:coverage_content_type = "physicalMeasurement" ;
+		PSAL:coordinates = "time lon lat depth" ;
+		PSAL:comment = "Practical salinity computed using UNESCO 1983 algorithm" ;
+        PSAL:sensor = "/CTD";
+	byte PSAL_QC(N_MEASUREMENTS) ;
+		PSAL_QC:_FillValue = 0b ;
+		PSAL_QC:long_name = "quality flag" ;
+	float CHLA(N_MEASUREMENTS) ;
+		CHLA:_FillValue = NaN ;
+		CHLA:long_name = "Chlorophyll-a concentration" ;
+        CHLA:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/CHLA/";
+		CHLA:standard_name = "mass_concentration_of_chlorophyll_a_in_sea_water" ;
+		CHLA:units = "mg m-3" ;
+		CHLA:ancillary_variables = "CHLA_QC" ;
+		CHLA:coverage_content_type = "physicalMeasurement" ;
+		CHLA:coordinates = "time lon lat depth" ;
+		CHLA:comment = "In-situ fluorometer with either manufacturer, laboratory or sample calibration applied" ;
+        CHLA:sensor = "/FLUOROMETER_CHLA";
+	byte CHLA_QC(N_MEASUREMENTS) ;
+		CHLA_QC:_FillValue = 0b ;
+		CHLA_QC:long_name = "quality flag" ;
+	float DOXY(N_MEASUREMENTS) ;
+		DOXY:_FillValue = NaN ;
+		DOXY:long_name = "Dissolved oxygen" ;
+        DOXY:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/DOXY/";
+		DOXY:standard_name = "moles_of_oxygen_per_unit_mass_in_sea_water" ;
+		DOXY:units = "micromol kg-1" ;
+		DOXY:ancillary_variables = "DOXY_QC" ;
+		DOXY:valid_max = 0. ;
+		DOXY:valid_min = 500. ;
+		DOXY:coverage_content_type = "physicalMeasurement" ;
+		DOXY:comment = "Concentration of dissolved oxygen per unit mass of the water column. Oxygen may be expressed in terms of mass, volume or quantity of substance" ;
+        DOXY:sensor = "/OPTODE_DOXY";
```

Comment:
This currently points to the old OPTODE_OXY group. Should point to the INSTRUMENT_OPTODE_DOXY variable, i.e.: DOXY:sensor = "INSTRUMENT_OPTODE_DOXY".

Reply:
@kerfoot I have updated the document accordingly